distributed/bokeh/tests/test_scheduler_bokeh.py

```diff
@@ -581,3 +581,37 @@ def test_root_redirect(c, s, a, b):
     )
     assert response.code == 200
     assert "/status" in response.effective_url
+
+
+@gen_cluster(
+    client=True,
+    scheduler_kwargs={"services": {("bokeh", 0): BokehScheduler}},
+    worker_kwargs={"services": {"bokeh": BokehWorker}},
+    timeout=180,
+)
+def test_proxy_to_workers(c, s, a, b):
+    dashboard_port = s.services["bokeh"].port
+    http_client = AsyncHTTPClient()
+    response = yield http_client.fetch("http://localhost:%d/" % dashboard_port)
+    assert response.code == 200
+    assert "/status" in response.effective_url
+
+    yield [a.heartbeat(), b.heartbeat()]
```

Comment:
Was this necessary?  If so, what data was missing?  We might want to make sure that this arrives on the first `register-worker` message.

Reply:
I was using this to ensure the workers were up before proceeding.  Apparently, it's not necessary

Reply:
Right, this is handled in `gen_cluster` for you.

Reply:
Also, the conventional way to wait for workers to be in a good state is just to yield/await them

```python
yield [a, b]
```

But again, we already do this, I just wanted to show off our `__await__` methods :)